masterlist.yaml

```diff
@@ -5012,6 +5012,74 @@ plugins:
   - name: 'Project Nevada - Med-X Fix.esp'
     tag: [ Names ]
 
+# Mothership Zeta Rehaul
+  - name: '(Mothership)?ZetaRehaul((BiWastelands|DelayDLCRedux|HardcoreZeta|HardcoreZetaDelayDLCRedux|NVSpeechChecks)Patch)?\.es[mp]'
+    url:
+      - link: 'https://www.nexusmods.com/newvegas/mods/77340/'
+        name: 'TTW - Mothership Zeta Rehaul'
+  - name: 'MothershipZetaRehaul.esm'
+    msg:
+      - <<: *patchProvided
+        subs: [ 'Tale of Bi Wastelands TTW' ]
+        condition: 'active("Tale of Bi Wastelands.esm") and not active("ZetaRehaulBiWastelandsPatch.esp")'
+      - <<: *patchProvided
+        subs: [ 'Delay DLC Redux TTW' ]
+        condition: 'active("DelayDLCRedux.esp") and not active("HardcoreZeta.esp") and not active("ZetaRehaulDelayDLCReduxPatch.esp")'
+      - <<: *patchProvided
+        subs: [ 'TTW Hardcore Zeta' ]
+        condition: 'active("HardcoreZeta.esp") and not active("DelayDLCRedux.esp") and not active("ZetaRehaulHardcoreZetaPatch.esp")'
+      - <<: *patchProvided
+        subs: [ 'Delay DLC Redux Hardcore Zeta' ]
+        condition: 'active("HardcoreZeta.esp") and active("DelayDLCRedux.esp") and not active("ZetaRehaulHardcoreZetaDelayDLCReduxPatch.esp")'
+      - <<: *patchProvided
+        subs: [ 'TTW New Vegas Speech Checks' ]
+        condition: 'active("TTW New Vegas Speech Checks.esm") and not active("ZetaRehaulNVSpeechChecksPatch.esp")'
```

Comment:
So when ordering messages, I would go by this order.
Sort them by alias first (*patchProvided, useVersion etc.)
And then order by what the subs contain. (subs: [ 'Delay DLC Redux TTW' ]).
I find it's much easier to comprehend and speeds up time to make changes and review.

```suggestion
    msg:
      - <<: *patchProvided
        subs: [ 'Delay DLC Redux TTW' ]
        condition: 'active("DelayDLCRedux.esp") and not active("HardcoreZeta.esp") and not active("ZetaRehaulDelayDLCReduxPatch.esp")'
      - <<: *patchProvided
        subs: [ 'Tale of Bi Wastelands TTW' ]
        condition: 'active("Tale of Bi Wastelands.esm") and not active("ZetaRehaulBiWastelandsPatch.esp")'
      - <<: *patchProvided
        subs: [ 'TTW Hardcore Zeta' ]
        condition: 'active("HardcoreZeta.esp") and not active("DelayDLCRedux.esp") and not active("ZetaRehaulHardcoreZetaPatch.esp")'
      - <<: *patchProvided
        subs: [ 'TTW Hardcore Zeta & Delay DLC Redux' ]
        condition: 'active("HardcoreZeta.esp") and active("DelayDLCRedux.esp") and not active("ZetaRehaulHardcoreZetaDelayDLCReduxPatch.esp")'
      - <<: *patchProvided
        subs: [ 'TTW New Vegas Speech Checks' ]
        condition: 'active("TTW New Vegas Speech Checks.esm") and not active("ZetaRehaulNVSpeechChecksPatch.esp")'
```